hbase-server/src/main/java/org/apache/hadoop/hbase/master/ServerManager.java

```diff
@@ -573,6 +574,7 @@ public synchronized long expireServer(final ServerName serverName) {
   }
 
   synchronized long expireServer(final ServerName serverName, boolean force) {
+    ZKWatcher zkw = master.getZooKeeper();
```

Comment:
We can include this inline directly on L591: `ZNodePaths.joinZNode(master.getZooKeeper().getZNodePaths().drainingZNode`....
Because for below 2 return statements, this just turns out to be redundant statement.

Reply:
done thx